server/fileupdater.mjs

```diff
@@ -261,3 +263,11 @@ function v3RemoveComputeAndRandomAndApplyVariables(routine) {
   for(const o of operationsToSplice.sort((a,b)=>a.index==b.index?b.order-a.order:b.index-a.index))
     routine.splice(o.index, 0, o.operation);
 }
+
+function v4ModifyDropTargetEmptyArray(properties) {
+  for(const property in properties) {
+    if(property == 'dropTarget')
+      if(Array.isArray(properties[property]) && properties[property].length == 0)
+        properties[property] = {};
+  }
```

Comment:
No need to loop through the properties. Just change `properties.dropTarget` directly.